docs/includes/pricing.md

```diff
@@ -116,3 +116,23 @@ Assuming both buy and sell are using market orders, a configuration similar to t
 ```
 
 Obviously, if only one side is using limit orders, different pricing combinations can be used.
+
+### Futures market's order pricing
+
+When trading on futures market. orderbook must be used because there is no ticker data for futures.
```

Comment:
This is not correct in general - but only valid for binance futures (as their futures ticker data does not contain prices).

If any, it should therefore be moved to the `exchanges.md` to the binance section - with a specific "futures" reference / subtitle...